city_scrapers/spiders/losca_City_Planning.py

```diff
@@ -0,0 +1,106 @@
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import BOARD, COMMISSION, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class LoscaCityPlanningSpider(CityScrapersSpider):
+    name = "losca_City_Planning"
+    agency = "Los Angeles City Planning"
+    timezone = "America/Los_Angeles"
+
+    start_urls = [
+        "https://planning.lacity.gov/dcpapi2/meetings/api/all/commissions/{start_year}",
+        "https://planning.lacity.gov/dcpapi2/meetings/api/all/boards/{start_year}",
+        "https://planning.lacity.gov/dcpapi2/meetings/api/all/hearings/{start_year}",
+    ]
+
+    meetings_url = "https://planning.lacity.gov/about/commissions-boards-hearings"
+
+    custom_settings = {
+        "RETRY_TIMES": 5,
+        "RETRY_HTTP_CODES": [429, 500, 502, 503, 504],
+        "DOWNLOAD_DELAY": 2,
+    }
+
+    def start_requests(self):
+        """
+        This spider retrieves meetings for the specified
+        `start_year`, which is dynamically set to the
+        current year.
+        """
+        current_year = datetime.now().year
+        for url in self.start_urls:
+            url = url.format(start_year=current_year)
+            yield scrapy.Request(
+                url=url,
+                headers={"Accept": "application/json"},
+                callback=self.parse,
+            )
+
+    def parse(self, response):
+        try:
+            data = response.json()
+            if "Entries" not in data:
+                self.logger.error("Missing 'Entries' field in response")
+                return
+            items = data["Entries"]
+        except ValueError as e:
+            self.logger.error(f"Failed to parse JSON response: {e}")
+            return
+
+        for item in items:
+            if not all(key in item for key in ["Type", "Date", "Address"]):
+                self.logger.warning(
+                    f"Skipping item due to missing required fields: {item}"
+                )
+                continue
+            meeting = Meeting(
+                title=item.get("Type", ""),
+                description=item.get("Note", "").strip(),
+                classification=self._parse_classification(item),
+                start=parse(item.get("Date")),
```

Comment:
<img width="435" alt="Screen Shot 2025-01-09 at 2 16 45 PM" src="https://github.com/user-attachments/assets/afddd891-8c83-4c4e-9a16-c0771383c53c" />

I see that all meetings get a start time of `00:00`. I see that the start times can be found in the linked PDFs. They seem to follow a pattern. Maybe we can have a default time per region? If there is not a pattern, maybe just add in the time notes to check the PDF for meeting time?